pkgs/misc/emulators/dolphin-emu/default.nix

```diff
@@ -1,13 +1,17 @@
-{ stdenv, pkgconfig, cmake, bluez, ffmpeg, libao, mesa, gtk2, glib
-, gettext, libpthreadstubs, libXrandr, libXext, readline
-, openal, libXdmcp, portaudio, SDL, wxGTK30, fetchurl
+{ stdenv, gcc5, pkgconfig, cmake, bluez, ffmpeg, libao, mesa, gtk2, glib
```

Comment:
It's preferred to not have the compiler be a dependency. Instead, you should override "stdenv" in all-packages.nix

https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/all-packages.nix#L6635